aten/src/ATen/native/cuda/RowwiseScaledMM.cu

```diff
@@ -747,7 +747,7 @@ void dispatch_fp8_rowwise_kernel_on_input_dtypes(
   }
 }
 
-template <typename... Types>
+// template <typename... Types>
```

Comment:
this is intentional right? it makes sense to remove I think

Reply:
Yep, just a small cleanup, it still compiles and works as before.